druid/src/app.rs

```diff
@@ -187,10 +187,10 @@ impl<T: Data> AppLauncher<T> {
         #[cfg(not(target_arch = "wasm32"))]
         {
             use tracing_subscriber::prelude::*;
-            let fmt_layer = tracing_subscriber::fmt::layer().with_target(true);
-            let filter_layer = tracing_subscriber::EnvFilter::try_from_default_env()
-                .or_else(|_| tracing_subscriber::EnvFilter::try_new("debug"))
-                .expect("Failed to initialize tracing subscriber");
+            let filter_layer = tracing_subscriber::filter::LevelFilter::DEBUG;
```

Comment:
I would disable the filter all together. This can be done via feature flags on the application side.

Reply:
With no filters the output is way too verbose. This is a good compromise for a default. If the user wants to fine-tune filter they can implement their own subscriber.